src/Talis/Persona/Client/Base.php

```diff
@@ -1,7 +1,10 @@
 <?php
 namespace Talis\Persona\Client;
+require __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
Well, a few things - a dependence on composer, for one.  A dependence on composer being in that location, for two. The other thing is that the class shouldn't care where its dependencies are?  That should be handled by whatever is using it.